config/detekt/baseline.xml

```diff
@@ -3,6 +3,8 @@
   <ManuallySuppressedIssues>
     <ID>ReturnCount:StepQuizReplyValidator.kt$StepQuizReplyValidator$@Suppress("CyclomaticComplexMethod") fun validate(reply: Reply, stepBlockName: String): ReplyValidationResult</ID>
     <ID>CyclomaticComplexMethod:StepQuizReplyValidator.kt$StepQuizReplyValidator$fun validate(reply: Reply, stepBlockName: String): ReplyValidationResult</ID>
+    <ID>TooManyFunctions:StepReducer.kt$StepReducer : StateReducer</ID>
+    <ID>LongMethod:StepDelegate.kt$StepDelegate$fun onAction( mainScreenRouter: MainScreenRouter, action: StepFeature.Action.ViewAction )</ID>
```

Comment:
We should add issues to the `ManuallySuppressedIssues` only if we are not going to fix them. I guess these issues we can fix. So let's add them to the `CurrentIssues`